sources/packages/web/src/composables/useApplication.ts

```diff
@@ -50,10 +50,7 @@ export function useApplication() {
       "Application #": application.applicationNumber ?? "-",
       Institution: application.applicationInstitutionName ?? "-",
       "Study dates": studyDates ?? "-",
-      Type:
-        application.applicationOfferingIntensity ??
-        application.data.howWillYouBeAttendingTheProgram ??
-        "-",
+      Type: application.applicationOfferingIntensity ?? "-",
```

Comment:
There is no scenario where the application.applicationOfferingIntensity will not be present, hence the null propagation "??" is no longer required.